miqa/core/models/evaluation.py

```diff
@@ -5,7 +5,12 @@
 
 class Evaluation(models.Model):
     id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
-    frame = models.ForeignKey('Frame', null=False, on_delete=models.PROTECT)
+    frame = models.OneToOneField(
+        'Frame',
+        null=False,
```

Comment:
```suggestion
```

This is the default value, and should kind of be our mental default as well; nullability is the exception to the rule, so I'd prefer to only specify the argument when it's `True`.

Reply:
Good point!